vulture_os/gui/static/js/listener.js

```diff
@@ -545,15 +548,26 @@ $(function() {
   }
 
   $('#id_filebeat_module').on("change", function(e) {
+    show_filebeat_input($('#id_mode').val(), $(this).val());
+    $('#id_filebeat_config').trigger('change');
     refresh_filebeat_module();
```

Comment:
put the trigger in the `refresh_filebeat_module() function, **after** the modification of the id_filebeat_config value

Reply:
done